python/python-core/playing-with-time/playing-with-time-discussion.md

```diff
@@ -0,0 +1,113 @@
+---
+author: Stefan-Stojanovic
+
+tags:
+  - discussion
+
+type: normal
+
+category: discussion
+
+---
+
+# Test Yourself
+
+---
+
+## Content
+
+Now that you've learned about time and date modules, it's time to practice!
+
+How would you create a program where you take a date using a module and separately print out all the different formats it provides?
```

Comment:
Should it be "take" a date or maybe "create" a date, using the "datetime" module?

```suggestion
How would you create a program where you create a date using the `datetime` module and separately print out all the different formats it provides?
```

To me, "take" primarily implies creating a function that takes arguments, not necessarily taking a date using a module.